pkg/logql/log/pipeline.go

```diff
@@ -139,3 +158,11 @@ func ReduceStages(stages []Stage) Stage {
 		return line, true
 	})
 }
+
+func unsafeGetBytes(s string) []byte {
```

Comment:
Q: so the basic idea is to avoid extra allocation during string <--> []bytes conversion correct?

Reply:
Yes in some case the compiler can guess but not all the time.

Here we know we only read the passed in line, we never mutate it, so we can safely avoid this.